docs/development.md

```diff
@@ -128,8 +128,9 @@ You need a student account which can be created by instructors.
 
 ## Testing
 
-TEAMMATES automated testing requires Firefox or Chrome (works on Windows and OS X).
-It is recommended to use Firefox 46.0 as this is the browser used in CI build (Travis/AppVeyor).
+* TEAMMATES automated testing requires Firefox or Chrome (works on Windows and OS X).
```

Comment:
Let's remove the OS supported since the major OSes are all supported.

Reply:
😄 was wondering why it was there

Reply:
Did you miss this?

Reply:
Checking with you if you missed this?